presto-yarn-package/src/main/slider/package/templates/queues.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+  "queues": {
+    "global": {
+      "maxConcurrent": 10,

```

Comment:
Yes ,It looks good to move those properties in app-config. Queues & Rules will not be  only "global", so , I just define one queue & rule.